### PR TITLE
feat: Add support for more conventional commit patterns

### PR DESCRIPTION
**Pull request by nhhagen**
Dependabot is also using `build(deps)` and `build(deps-dev)` for
dependency updates.

### CODE REVIEW

**Review by bendiknesbo — DISMISSED** [review]

File: .github/release-drafter.yml [hunk 1]
```diff
@@ -71,7 +71,7 @@ autolabeler:
   # labels to help with changelog grouping
   - label: "release-drafter:dependency"
     title:
-      - '^(?:Bump|(?:chore\(deps\):|chore\(deps-dev\)):)' # Dependency update PR by Dependabot
+      - '^(?:Bump|(?:chore\(deps\)|chore\(deps-dev\)|build\(deps\)|build\(deps-dev\)):)' # Dependency update PR by Dependabot
```
Comment (bendiknesbo) [inline .github/release-drafter.yml:74]: Consider simplifying to: `^(?:Bump|chore\(deps\)|chore\(deps-dev\)|build\(deps\)|build\(deps-dev\))`

Comment (nhhagen) [reply .github/release-drafter.yml:74]: That misses the `:` after the conventional commit prefixes

PS: The previous pattern had a bug...

Comment (nhhagen) [reply .github/release-drafter.yml:74]: But there is another simpler one, give me a few minutes

Comment (nhhagen) [reply .github/release-drafter.yml]: `/^(?:Bump|(?:chore|build)\((?:deps|deps-dev)\):)\s.*/i` should be simpler and align better with the other implementations

**Review by bendiknesbo — APPROVED** [review]

**Review by pravindahal — APPROVED** [review]

**Pull request merged**